### PR TITLE
Move network utils that are used by AWS modules

### PR DESCRIPTION
**Pull request by s-hertel**
##### SUMMARY
Separate networking tools that may be used by modules outside of networking so changes to networking-only utilities don't trigger AWS integration tests

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
lib/ansible/module_utils/common/network.py

##### ANSIBLE VERSION
```
ansible 2.8.0.dev0
```

### CODE REVIEW

File: lib/ansible/module_utils/network/common/utils.py [hunk 1]
```diff
@@ -444,104 +445,9 @@ def contains_vars(self, data):
...
-
-
-def is_masklen(val):
-    try:
-        return 0 <= int(val) <= 32
-    except ValueError:
-        return False
-
-
 def to_bits(val):
```
Comment (mattclay) [inline lib/ansible/module_utils/network/common/utils.py]: Although only used by the network modules, it isn't specific to them, so this should probably move as well.

**Review by mattclay — CHANGES_REQUESTED** [review]

**Review by ryansb — APPROVED** [review]

**Review by mattclay — APPROVED** [review]

**Review by Qalthos — APPROVED** [review]

**Comment by pabelanger** [comment]
I'll look into this

**Review by pabelanger — APPROVED** [review]
+1 looks correct

**Comment by s-hertel** [comment]
rebuild_merge

**Pull request merged**